boxoffice/views/utils.py

```diff
@@ -24,6 +23,12 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def check_api_access(access_token):
+    """Aborts if a request does not have the correct access token"""
+    if not request.args.get('access_token') or request.args.get('access_token') != access_token:
```

Comment:
This choice of parameter name adds overhead in Flask-Lastuser, which also attempts to check the access token. https://github.com/hasgeek/flask-lastuser/blob/master/flask_lastuser/__init__.py#L130